test/Java9andUp/playlist.xml

```diff
@@ -229,7 +229,40 @@
 		<subsets>
 			<subset>SE90</subset>
 		</subsets>
-	</test>	
+	</test>
+	<test>
+		<testCaseName>StaticAccessChecks-IllegalAccessPermit</testCaseName>
+			<command>$(JAVA_COMMAND) $(JVM_OPTIONS) --illegal-access=permit\
+     -cp $(Q)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)GeneralTest.jar$(P)$(LIB_DIR)$(D)asm-all.jar$(Q) \
+     org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) \
+     -testnames DenyAccess \
```

Comment:
arguments for -testnames is a comma separated list of test names, not classname. In this case, we should use AccessUnRestrictedClass. Same for the PermitAccess test below, AccessRestrictedClass should be used.